test/helpers/wait-for-update.ts

```diff
@@ -12,6 +12,7 @@ import Vue from 'vue'
 //   // more assertions...
 // })
 // .then(done)
+// @ts-expect-error
 window.waitForUpdate = initialCb => {
```

Comment:
Here we should be able to add `waitForUpdate` on the window object, removing the need to `//@ts-expect-error`